test/test_trigger.py

```diff
@@ -80,6 +80,56 @@ def test_quit_pretrigger(nick):
     assert pretrigger.sender is None
 
 
+def test_quit_pretrigger_empty(nick):
+    """Make sure empty quit message is also fine"""
+    line = ':Foo!foo@example.com QUIT'
+    pretrigger = PreTrigger(nick, line)
+    assert pretrigger.tags == {}
+    assert pretrigger.hostmask == 'Foo!foo@example.com'
+    assert pretrigger.line == line
+    assert pretrigger.args == []
+    assert pretrigger.text == 'QUIT'  # TODO BUG: the command is not part of `pretrigger.args`
```

Comment:
Nope, I discovered it while working on this PR and created a follow-up immediately: #2360.

Reply:
I'm a big fan of leaving an explicit issue reference with a comment like this, but as long as it's tracked, it's not a big deal.

Reply:
I would be, too, except in cases like this where such an issue reference doesn't yet exist. Creating an issue instead of just making the +1-1 patch immediately seemed silly. (OK, the actual patch is +5-4, but that's because of updating comments + tests.)